src/backend/src/services/design-reviews.services.ts

```diff
@@ -158,11 +158,16 @@ export default class DesignReviewsService {
       throw new NotFoundException('User Settings', 'Cannot find settings of members');
     }
 
+    console.log(memberUserSettings);
     // send a slack message to all leadership invited to the design review
     for (const memberUserSetting of memberUserSettings) {
       if (memberUserSetting.slackId) {
         try {
-          await sendSlackDesignReviewNotification(memberUserSetting.slackId, designReview.designReviewId);
+          await sendSlackDesignReviewNotification(
```

Comment:
We should also do this on edit, right?

Reply:
We need to figure out how adding people post a design review getting confirmed is going to work, I think we'll do that post-release tbh